backend/database/README.md

```diff
@@ -1,33 +1,79 @@
+- For database [make recipes](../Makefile)
 
-### How to perform a database migration
+## How to perform a database migration
 
 1. From the top level directory (`corpora-data-portal`), `cd backend`.
 1. Run `make db/new_migration MESSAGE="purpose_of_migration"` where `purpose_of_migration` is a short phrase describing why the database migration is occurring.
 1. Head over to the file that was just created. The path will be something like `backend/database/versions/xxxxxxxxxxxx_purpose_of_migration.py`. Edit the `upgrade()` and `downgrade()` functions such that `upgrade()` contains the commands to perform the migration you would like and `downgrade()` contains the commands to undo it.
-1. Enable local connection to the private `dev` RDS instance through Bastion: `make db/tunnel`. See [Connect to Remote RDS](#connect_to_remote_rds) for more details.
-1. Once you've completed the changes (and at this point, you can create a PR to get the functions reviewed), test your migration. Ensure that you are [connected](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/chalice/api_server/README.md#development) to the database you'd like to migrate (usually `TEST`). Run `DEPLOYMENT_STAGE=test make db/migrate`.
-1. If no errors are outputted, complete the migration by running `DEPLOYMENT_STAGE=dev make db/migrate`. Ensure that tunneling is set up and `CORPORA_LOCAL_DEV` is set to true.
+1. [Test your migration](#test-a-migration)
+1. Once you've completed the changes, create a PR to get the functions reviewed. 
+1. Once the PR is merged, you can run the migration.
+1. [Connect to Remote RDS](#connect-to-remote-rds)
+1. In a new terminal, complete the migration by running:
+```shell
+cd corpora-data-portal/backend
+export CORPORA_LOCAL_DEV=1
+export DEPLOYMENT_STAGE=dev 
+make db/migrate
+```
 
-### How to autogenerate migration script
+## How to autogenerate migration script
 
 1. From the top level directory (`corpora-data-portal`), `cd backend`.
-1. Makes changes to [corpora_orm.py](../corpora/common/corpora_orm.py)
-1. Set DEPLOYMENT_STAGE=dev, CORPORA_LOCAL_DEV=1
-1. Run `make db/connect`
-1. Follow [How to perform a database migration](#how_to_perform_a_database_migration) instructions except use `make db/new_migration_auto MESSAGE="purpose_of_migration"`. See [What does Autogenerate Detect (and what does it not detect?)](https://alembic.sqlalchemy.org/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect).
+1. Make changes to [corpora_orm.py](../corpora/common/corpora_orm.py)
+1. [Connect to Remote RDS](#connect-to-remote-rds)
+1. Run Auto-migration:
+```shell
+cd corpora-data-portal/backend
+export CORPORA_LOCAL_DEV=1
+export DEPLOYMENT_STAGE=dev 
+make db/new_migration_auto MESSAGE="purpose_of_migration"
+```
+See [What does Autogenerate Detect (and what does it not detect?)](https://alembic.sqlalchemy.org/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect).
```

Comment:
Maybe add a note to compare the created file to the changes you've made to the orm to ensure they match. And explicitly call out that autogenerate doesnt handle name changes well?

Reply:
> Maybe add a note to compare the created file to the changes you've made to the orm to ensure they match.

Added a line to check that corpora_orm.py agrees with the change in the migration.

> And explicitly call out that autogenerate doesnt handle name changes well?

I'm going to leave it up the to reader to follow that link to check what alembic supports. Otherwise it might change under our noses since alembic is an active project.